### DISCUSSION TITLE
Is "viewer" parameter working?

### DISCUSSION THREAD

**Discussion by LITUATUI** [razonyang/hugo-theme-bootstrap#1031] [Q&A] [answered]
Hi.

I want to disable the [image gallery](https://hbs.razonyang.com/v1/en/docs/image-viewer/) and set on params.toml:

`viewer = false # Image Viewer
`

However, it doesn't seem to do anything, since the image gallery is still active.

**Comment by razonyang** [comment, marked as answer]
Yes, it does, just confirmed this on `exampleSite`.

A common case is that the `viewer` was put into wrong context when using TOML, for example,

```toml
[some_context]
// ...
viewer = false
// ...
```

**Reply by LITUATUI** [reply]
Thanks, it was exactly that.
